components/StoreListItem/index.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+const StyledListItem = styled.li`
+  margin-block: 7px;
+  width: 100%;
+  background-color: var(--secondaryBackgroundColor);
+  border-radius: 5px;
+  box-shadow: 0px 1px 3px var(--primaryDarkColor);
+`;
+
+const StyledCard = styled.span`
```

Comment:
Since a ``span`` element highlights a part of a inline element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span) I would recommend here to use a ``section`` element.

Reply:
Thanks, we changed it to a section.